changelog.md

```diff
@@ -1,3 +1,6 @@
+# 0.0.5
+  * Rename package to HPOBench
+
```

Comment:
There is probably a bit more:
  * Added BNN benchmark
  * Updated returned loss values for NB201